helm/bailo/templates/mail/mail.service.yaml

```diff
@@ -7,13 +8,14 @@ metadata:
     name: mail
 spec:
   ports:
-    - port: {{ .Values.config.smtp.port }}
-      targetPort: {{ .Values.config.smtp.port }}
+    - port: 1080
+      targetPort: 1080
       protocol: TCP
       name: http
-    - port: 1025
-      targetPort: 1025
+    - port: {{ .Values.config.smtp.port }}
+      targetPort: {{ .Values.config.smtp.port }}
```

Comment:
This target port will always be 1025 right?  It's encoded into the container we use.  I think it makes sense to configure the externally visible port (under .Values.config.smtp.port), but not the targetPort.

Reply:
I have configured .Values.config.smtp.port on mail.deployment.yaml

Reply:
I had configured mail.deployment.yaml to listen to {{ .Values.config.smtp.port }}, hence the mail.service.yaml reflects that